doc/source/manual/state_snapshot_transfer.rst

```diff
@@ -11,13 +11,12 @@ In order to be synchronized with the cluster,
 the new node has to receive data from a node
 that is already part of the cluster.
 
-There are three methods of SST available in |PXC|:
+There are two methods of SST available in |PXC|:
```

Comment:
rsync has also been removed from 8.0.  So there is only one sst method available, "xtrabackup"